.coveragerc.pytest

```diff
@@ -0,0 +1,8 @@
+[run]
```

Comment:
Is there any reason this file can't be named `.coveragerc`?

Reply:
Coverage is run with three different config files,
+ pytest      -  .coveragerc.pytest
+ ctest        -  {build}/girder_worker/core/tests/girder_worker.coveragerc
+ combine  -  no config file (i.e. defaults)

the pytest coverage ignores all files in girder_worker/core/ and girder_worker/plugins/. If you try and combine the pytest coverage (which ends up in .coverage.pytest)  with the ctest coverage (which ends up in .coverage.legacy) with the .coveragerc.pytest config file,  then it ignores everything the ctest coverage created.  

Eventually .coveragerc.pytest will become .coveragerc,  but not until core and plugins are transitioned out of girder_worker.